core/src/test/scala/org/typelevel/paiges/PaigesTest.scala

```diff
@@ -34,15 +34,15 @@ object PaigesTest {
 }
 
 class PaigesTest extends FunSuite {
-  import Doc.text
+  import Doc.{line, text}
```

Comment:
I'll undo this change.  It was a mixup on my part between the `Text` constructor, which can't have \n, and the `text` function that can.